turkce/darbe-kumpas/darbeler.md

```diff
@@ -0,0 +1,2 @@
+Su siteden butun darbeleri desifre edebilirsin ozellikle paralel olanalarini
+[star-darbeler](www.star.com.tr)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.